index.html

```diff
@@ -8,75 +8,89 @@
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#form-footer">Contact</a></li>
+          <li><a href="#"><img src="Assets/Letter.png" alt=""></a></li>
+      </ul>
+				<a class="icon-menu" href="#"><img src="Assets/Icon-Menu.png" alt=""></a>
 			</nav>
-	</header>
-
-		<div class="personal-info">
+	</header> 
+  
+  <section class="personal-info">
+    <div class="brief-intro">
 			<h1>Hey There. <br> I am Shakur</h1>
 			<h3>I am a Software Developer</h3>
 			<p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
 				don’t hestiate to contact me.</p>
+      </div>
 				<ul class="links">
 					<li class="lists"><a href="#"> <img src="Assets/github.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Linkedin.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/Vector.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/twitter.png" alt=""></a></li>
 					<li class="lists"><a href="#"> <img src="Assets/medium.png" alt=""></a></li>
 				</ul>
-		 
-		</div>
-		  <h2 class="recent">My Recent Works</h2>
-		<div class="works">
-			<div class="multi">
-				<h2 class="multi-post">Multi-Post Stories</h2>
-			<p>A daily selection of privately personalized reads;
-				 no accounts or sign-ups required. has been the 
-				 industry's standard dummy text ever since the 1500s, 
-				 when an unknown printer took a standard dummy text.</p>
-			<ul class="buttons">
-				<li class="btn-skills">css</li>
-				<li class="btn-skills">html</li>
-				<li class="btn-skills">bootstrap</li>
-				<li class="btn-skills">Ruby</li>
-			</ul>
-			<div class="btn-skls">
-        <button class="btn-skills-project">See Project</button>
-      </div>				
-      
-			</div>
-		</div>
+  </section>
 
-		<div class="data">
+  <section class="recent-profession">
+          <h2 class="recent-header">My Recent Works</h2>          
+      <div class="multi-title"></div>
+              
+    <div class="works">
+      <img src="Assets/backgroundcard.png" alt="placeholder" />
+      <div class="multi">
+        <h2 class="multi-post">Multi-Post Stories</h2>
+        <p>A daily selection of privately personalized reads;
+          no accounts or sign-ups required. has been the 
+          industry's standard dummy text ever since the 1500s, 
+          when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="buttons">
+          <li class="btn-skills">css</li>
+          <li class="btn-skills">html</li>
+          <li class="btn-skills">bootstrap</li>
+          <li class="btn-skills">Ruby</li>
+        </ul>
+        <div class="btn-skls">
+          <button class="btn-skills-project">See Project</button>
+        </div>        
+        </div>
+      </div>
+      
+    </section>
+    
+  <section class="skills-grid">

```

Comment:
Reduce the width of the section and replace the see project button with the languages
#### Original design 
![right3](https://user-images.githubusercontent.com/58376020/122623495-964c7200-d09c-11eb-996c-01f021eb6542.png)

#### Your Project
![wrong3](https://user-images.githubusercontent.com/58376020/122623515-a49a8e00-d09c-11eb-847f-03830ed8e557.png)

Reply:
## @Davidosky007,

Thanks for your review, I have been able to add proper project image and styling, below is a screenshot of how it is rendering on my desktop with the width of 30% using flex property. I would like clarification if there is anything I need to add more, I would like to implement all changes before I submit them for review as I only have one more review. 
Thanks.
![cardscreen](https://user-images.githubusercontent.com/61184208/122641295-4bfadd80-d0fc-11eb-9e32-4f12bccf93a2.png)